libs/SalesforceSDK/build.gradle.kts

```diff
@@ -13,20 +13,20 @@ plugins {
 dependencies {
     api(project(":libs:SalesforceAnalytics"))
     api("com.squareup.okhttp3:okhttp:4.10.0")
-    api("com.google.firebase:firebase-messaging:23.3.1")
-    api("androidx.core:core:1.12.0")
-    api("androidx.browser:browser:1.7.0")
+    api("com.google.firebase:firebase-messaging:24.0.0")
```

Comment:
@brandonpage do you think this one could be an issue?

Reply:
Doesn't look like it [from the changelog](https://firebase.google.com/support/release-notes/android#messaging_v24-0-0).